engine/unity5/Assets/Scripts/Wireframe.cs

```diff
@@ -0,0 +1,152 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class Wireframe : MonoBehaviour
+{
+
+    public bool render_mesh_normaly = true;
+    public bool render_lines_1st = false;
+    public bool render_lines_2nd = false;
+    public bool render_lines_3rd = false;
+    public Color lineColor = new Color(0.0f, 1.0f, 1.0f);
+    public Color backgroundColor = new Color(0.0f, 0.5f, 0.5f);
+    public bool ZWrite = true;
+    public bool AWrite = true;
+    public bool blend = true;
+    public float lineWidth = 4;
+    public int size = 0;
+
+    private Vector3[] lines;
+    private ArrayList lines_List;
+    public Material lineMaterial;
+    private Vector3 offset;
+    //private MeshRenderer meshRenderer; 
+
+    /*
+    ████████       ▄▀▀■  ▀▀█▀▀  ▄▀▀▄  █▀▀▄  ▀▀█▀▀ 
+    ████████       ▀■■▄    █    █■■█  █▀▀▄    █   
+    ████████       ■▄▄▀    █    █  █  █  █    █   
+    */
```

Comment:
That's some dope ASCII art right there. 👍

Reply:
Thanks to our friends at stack overflow